### DISCUSSION TITLE
[OP Stack Deployment] Unable to build any Docker image for optimism v1.2.0

### DISCUSSION THREAD

**Discussion by awhite206** [ethereum-optimism/developers#34] [Deploying the OP Stack] [answered]
### Issue Description

For latest version of optimism repo, v1.2.0, when I try to build the op-node docker image, I get an error about `"ERROR: failed to solve: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-stack-go:latest: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-stack-go:latest: not found"`

Any idea what's wrong?

Cmd I ran was:

`make devnet-up && docker build --output type=docker --tag 'ghrc.io/polymerdao/op-node:dev' --build-arg="OP_VERSION=1.20" --file op-node/Dockerfile .`

**Comment by opfocus** [comment]
This seems to be fixed in a new commit, please try the new commit   https://github.com/ethereum-optimism/optimism/commit/8358328132512ca974b5bd73dc181c1ca4d623fb

**Reply by awhite206** [reply]
Nope, still broken. I also notice in the new version that `make golang-docker` is broken, too.

```# We don't use a buildx builder here, and just load directly into regular docker, for convenience.
GIT_COMMIT=$(git rev-parse HEAD) \
	GIT_DATE=$(git show -s --format='%ct') \
	IMAGE_TAGS=$GIT_COMMIT,latest \
	docker buildx bake \
			--progress plain \
			--load \
			-f docker-bake.hcl \
			op-node op-batcher op-proposer op-challenger
ERROR: invalid tag "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-stack-go:": invalid reference format
```

And to be clear, building the docker image as above still gives me

```
ERROR: failed to solve: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-stack-go:latest: us-docker.pkg.dev/oplabs-tools-artifacts/images/op-stack-go:latest: not found
```

**Comment by ajsutton** [comment, marked as answer]
Yep, I can confirm this issue - the Makefile has a bug with how it sets the env vars.  Should be fixed by https://github.com/ethereum-optimism/optimism/pull/7971

**Reply by awhite206** [reply]
Thank you
